docusaurus.config.js

```diff
@@ -349,6 +349,52 @@ const config = {
             from: "/getting-started/connecting-your-app",
             to: "/overview/connecting-your-app",
           },
+          {
+            from: "/category/supported-sdks",
+            to: "/category/-sdks",
```

Comment:
update this- I merged the sdks page it's now - 

https://docs.permit.io/sdk/sdks-overview